.changeset/eight-planes-design.md

```diff
@@ -0,0 +1,49 @@
+---
+"thirdweb": minor
+---
+
+Adds basic EIP-5792 support for wallets with functions `getCapabilities`, `sendCalls`, `showCallsStatus`, `getCallsStatus`.
+
+## Example Usage
+
+### `getCapabilities`
+
+Returns the capabilities of the wallet according to EIP-5792.
+
+```ts
+const capabilities = await getCapabilities({ wallet });
+```
+
+### `sendCalls`
+
+Sends the given calls to the wallet for execution, and attempts to fallback to normal execution if the wallet does not support EIP-5792.
+
+```ts
+const preparedTx = approve({
+  contract: USDT_CONTRACT,
+  amount: 100,
+  spender: TEST_ACCOUNT_B.address,
+});
+
+const bundleId = await sendCalls({
+  wallet,
+  client,
+  calls: [preparedTx],
+});
+```
+
+### `showCallsStatus`
+
+Requests the wallet to show the status of a given bundle ID.
+
+```ts
+await showCallsStatus({ wallet, bundleId });
```

Comment:
does this actually work? it opens the wallet with the current status?

Reply:
It works *if the wallet supports 5792*. I'm honestly not a big fan of it since it makes app developers more reliant on the wallet, but its in the spec so I included it.